scripts/initialize-build-host.sh

```diff
@@ -315,6 +315,11 @@ then
                 ;;
         esac
     done > env.sh
+
+    # make it easy to check if running in a proxied target
+    echo "PROXIED=1" >> env.sh
```

Comment:
in other scripts I saw checking for `proxy-target.txt` file, but I don't mind against an extra way to check this :-)

Reply:
I prefer this explicit way.